Readme/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Achtung!!
+Die Verwendung von Daikin-Rotex-HPSU-CAN kann potenziell Ihr Heizsystem beschädigen. Die Nutzung erfolgt auf eigene Verantwortung. Ich übernehme keine Haftung für entstandene Schäden.
+
+## Anleitung: Installation ESPHome
+
+Diese Anleitung hilft dir, ESPHome auf Linux/MacOS oder Windows zu installieren, das Dashboard zu nutzen und ein ESP32-Gerät für die Home Assistant-Steuerung deiner Rotex/Daikin HPSU Compact einzurichten.
+
+### 1. Verwendung ESPHome Manager
+
+1. Du lädst dir je nach Betriebssystem diese Datei auf deinen Computer herunter.
+   - Windows: https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/tree/dev/scripts/ManageMinicondaESPHome_Win.ps1
+   - Linux/MAC: https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/tree/dev/scripts/ManageESPHome_Linux_MacOS.sh
+
+| **Windows**                                           | **Linux/MacOS**                                   |
+|------------------------------------------------------|--------------------------------------------------|
+| In einer Admin PowerShell diesen Befehl ausführen.<br>`PowerShell -ExecutionPolicy Bypass -File "X:\Path\To\InstallMinicondaESPHome.ps1"` | In einem Terminal/Console diese Befehle ausführen.<br>`chmod +x /Path/To/ManageESPHome_Linux_MacOS.sh`<br><br>`./Path/To/ManageESPHome_Linux_MacOS.sh` |
```

Comment:
ManageESPHome_Linux_MacOS.sh need also to be changed

Reply:
Added or changed accordingly in last commit.